ambient/foundation-sites.json

```diff
@@ -0,0 +1,5 @@
+{
+  "versions": {
+    "6.1.0": "github:samvloeberghs/foundation-sites-typings/index.d.ts#ddfa8f2eff7770d5cbb67c86557a660aee2c8456"
```

Comment:
I would just make this `github:samvloeberghs/foundation-sites-typings#ddfa8f2eff7770d5cbb67c86557a660aee2c8456` (without `index.d.ts`) so it'll resolve the `typings.json`. However, since `jquery` is an ambient definition in `typings.json`, you'll need to rename `dependencies` to `ambientDependencies` in `typings.json`. Does that make sense? I'm trying to find an easier way to explain and show this for people that aren't me, so apologies in advance :smile: